cleanrl/ddpg_continuous_action.py

```diff
@@ -1,3 +1,6 @@
+# docs and experiment results can be found at
+# https://docs.cleanrl.dev/rl-algorithms/ddpg/#ddpg_continuous_actionpy
```

Comment:
Same question for the link here

Reply:
https://docs.cleanrl.dev/rl-algorithms/ppo/#ppo_continuous_actionpy works for example.